Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:alpine3.19@sha256:0466223b8544fb7d4ff04748acc4d75a608234bf4e79563bff208d2060c0dd79
+FROM golang:1.23.1-alpine3.20@sha256:ac67716dd016429be8d4c2c53a248d7bcdf06d34127d3dc451bda6aa5a87bc06
```

Comment:
Can we include this change in another PR? I think it's a good change.

Reply:
https://github.com/stacklok/frizbee-action/pull/55